package.json

```diff
@@ -1,157 +1,157 @@
+    "azure-arm-cognitiveservices": "1.0.0-preview",
+    "azure-arm-commerce": "1.0.0-preview",
+    "azure-arm-compute": "1.0.0-preview",
+    "azure-arm-containerregistry": "1.0.0-preview",
+    "azure-arm-customerinsights": "1.0.0-preview",
+    "azure-arm-datalake-analytics": "2.0.0-preview",
+    "azure-arm-datalake-store": "2.0.0-preview",
+    "azure-arm-devtestlabs": "1.0.0-preview",
+    "azure-arm-dns": "1.0.0-preview",
+    "azure-arm-documentdb": "1.0.0-preview",
+    "azure-arm-eventhub": "1.0.0-preview",
+    "azure-arm-hdinsight": "0.2.1",
+    "azure-arm-hdinsight-jobs": "0.1.1",
+    "azure-arm-insights": "1.0.0-preview",
+    "azure-arm-intune": "1.0.0-preview",
+    "azure-arm-iothub": "1.0.0-preview",
+    "azure-arm-keyvault": "1.0.0-preview",
+    "azure-arm-logic": "1.0.0-preview",
+    "azure-arm-machinelearning": "1.0.0-preview",
+    "azure-arm-mediaservices":"1.0.0-preview",
+    "azure-arm-monitor": "0.16.1-preview",
+    "azure-arm-network": "1.0.0-preview",
+    "azure-arm-notificationhubs": "1.0.0-preview",
+    "azure-arm-operationalinsights":"1.0.0-preview",
+    "azure-arm-powerbiembedded": "1.0.0-preview",
+    "azure-arm-recoveryservicesbackup": "1.0.0-preview",
+    "azure-arm-recoveryservices": "1.0.0-preview",
+    "azure-arm-rediscache": "1.0.0-preview",
+    "azure-arm-relay": "1.0.0-preview",
+    "azure-arm-resource": "2.0.0-preview",
+    "azure-arm-sb": "1.0.0-preview",
+    "azure-arm-scheduler": "1.0.0-preview",
+    "azure-arm-search" :"1.0.0-preview",
+    "azure-arm-servermanagement": "1.0.0-preview",
+    "azure-arm-servicefabric": "1.0.0-preview",
+    "azure-arm-servicemap": "1.0.0-preview",
+    "azure-arm-sql": "1.0.0-preview",
+    "azure-arm-storage": "1.0.0-preview",
+    "azure-arm-storageimportexport": "1.0.0-preview",
+    "azure-arm-trafficmanager": "1.0.0-preview",
+    "azure-arm-website": "1.0.0-preview",
+    "azure-batch": "1.0.0-preview",
+    "azure-common": "0.9.18",
+    "azure-gallery": "2.0.0-pre.20",
+    "azure-graph": "2.1.0-preview",
+    "azure-insights": "1.0.0-preview",
+    "azure-keyvault": "2.0.0-preview",
+    "azure-monitor": "0.16.1-preview",
+    "azure-monitoring": "0.10.5",
+    "azure-sb": "0.10.4",
+    "azure-servicefabric": "1.0.0-preview",
+    "azure-scheduler": "0.10.3",
+    "azure-storage": "1.3.0",
+    "mime": "~1.2.4",
+    "ms-rest-azure": "^2.0.1",
+    "ms-rest": "^2.0.0",
+    "moment": "^2.18.1",
+    "underscore": "1.4.x",
+    "request": "^2.81.0",
+    "uuid": "^3.0.1"
+  },
+  "devDependencies": {
+    "adal-node": "^0.1.22",
+    "azure-storage-legacy": "0.9.14",
+    "glob": "^7.1.1",
+    "grunt": "~0.4",
+    "grunt-contrib-connect": "^0.10.1",
+    "grunt-contrib-symlink": "^0.3.0",
+    "grunt-devserver": "~0.6",
+    "grunt-jsdoc": "~0.6",
+    "grunt-gh-pages": "1.1.0",
+    "jshint": "2.9.4",
+    "minami": "git://github.com/devigned/minami#master",
+    "mocha": "^3.2.0",
+    "nock": "^9.0.14",

```

Comment:
dont change the nock version for rollup azure.
Instead add  nock as a dev dependency in ms-rest-azure's package.json and this test over there. Since this is more of a unit test and not a scenario/live test anymore.